packages/inventory/src/EntityTable.js

```diff
@@ -63,7 +63,11 @@ class EntityTable extends React.Component {
             }
 
             if (isTime) {
-                const [ day, date, month, year, time ] = new Date(get(col, key, ' ')).toUTCString().split(' ');
+                if (DateFormat) {
+                    return { title: <DateFormat date={get(col, key, ' ')} type="relative" /> };
+                }
+
+                const [ , date, month, year, time ] = new Date(get(col, key, ' ')).toUTCString().split(' ');
```

Comment:
is leading comma intentional? 🤔

Reply:
Yup, that's to omit first element from array.